[fix] indentation for hypothesis profile and proper inheritance for QuantizationTestCase

jerryzh168 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#25934 [fix] indentation for hypothesis profile and proper inheritance for QuantizationTestCase**

Summary:
att

Test Plan:
python test test/test_quantized_nn_mods.py

Reviewers:
pt1quant
Subscribers:

Tasks:

Tags:

Differential Revision: [D17318270](https://our.internmc.facebook.com/intern/diff/D17318270)

jerryzh168 committed:
[fix] indentation for hypothesis profile and proper inheritance for QuantizationTestCase

Summary:
att

Test Plan:
python test test/test_quantized_nn_mods.py

Reviewers:
pt1quant
Subscribers:

Tasks:

Tags:

bddppq commented on test/common_quantization.py:
```diff
@@ -60,7 +60,11 @@ def prepare_dynamic(model, qconfig_dict=None):
 
 # QuantizationTestCase used as a base class for testing quantization on modules
 class QuantizationTestCase(TestCase):
+    def __init__(self, method_name='runTest'):
```
This shouldn't be needed

jerryzh168 committed:
Update on "[fix] indentation for hypothesis profile and proper inheritance for QuantizationTestCase"


Summary:
att

Test Plan:
python test test/test_quantized_nn_mods.py

Reviewers:
pt1quant
Subscribers:

Tasks:

Tags:

jerryzh168 committed:
Update on "[fix] indentation for hypothesis profile and proper inheritance for QuantizationTestCase"


Summary:
att

Test Plan:
python test test/test_quantized_nn_mods.py

Reviewers:
pt1quant
Subscribers:

Tasks:

Tags:

bddppq reviewed (approved):
LGTM

ezyang commented:
How was this tested?

jerryzh168 commented:
@ezyang tested locally by enabling pytorch_ci profile and printing derandomize variable in the test case
